lib/premailer/rails/css_loaders/asset_pipeline_loader.rb

```diff
@@ -5,18 +5,10 @@ module AssetPipelineLoader
         extend self
 
         def load(url)
-          if asset_pipeline_present?
-            file = file_name(url)
-            asset = ::Rails.application.assets.find_asset(file)
-            asset.to_s if asset
-          end
-        end
-
-        def asset_pipeline_present?
-          defined?(::Rails) &&
-            ::Rails.respond_to?(:application) &&
-            ::Rails.application.respond_to?(:assets) &&
-            ::Rails.application.assets
```

Comment:
The reason why we were doing `defined?(::Rails)` is to make this gem work in the absence of Rails (yes, I know, the name suggests otherwise 😄). From the README:

> Although premailer-rails has certain Rails specific features, *it also works in the absence of Rails* making it compatible with other frameworks such as sinatra.

I know of some people who use it with just Action Mailer outside of Rails. Can we thus keep the assumption that `::Rails` might not be defined? Thanks! 😊

Reply:
Good point, thanks for the context.